pkgs/data/icons/gruvbox-dark-icons-gtk/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-icons-gtk";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fks2rrrb62ybzn8gqan5swcgksrb579vk37bx4xpwkc552dz2z2";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
+    rm README.md
+    cp -r ${src} $out/share/icons/${pname}
+  '';
+
+  dontDropIconThemeCache = true;
+
+  meta = with stdenv.lib; {
+    description = "Gruvbox icons for GTK based desktop environments";
+    homepage = "https://github.com/jmattheis/gruvbox-dark-gtk";
+    license = licenses.gpl3;
+    platforms = platforms.unix;
+    manitainers = [ maintainers.nomisiv ];
```

Comment:
You need to add yourself as a maintainer in the [maintainer-list.nix](https://github.com/NixOS/nixpkgs/blob/c44acaaceb04ab149b805656eeffcdf2acf939e6/maintainers/maintainer-list.nix) file for this to work.

Please do so in a separate commit (`maintainers: add nomisiv`).